### PR TITLE
Create contact structure

### LINKED ISSUE #9: Create contact page structure
- Flexbox 
- 1 row, 2 columns

### PR DESCRIPTION
**Pull request by PAZHames**
- added utility classes for flex-col and flex-row
- created structure for contact - form to go on right, contact details etc on left 
- closes #9 

also, my terminal is absolutely refusing to link to github, so it's all being done through github desktop - do let me know if anything is off!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,16 @@ <h2>Meet the Team</h2>
 
     <section id="contact">
         <h2>Contact</h2>
+        <div class="flex-row">
+            <div class="felx-col">
```
Comment (Frannerz) [inline index.html]: Think there is a typo here with class name

**Review by Frannerz — CHANGES_REQUESTED** [review]
All good just spotted a small typo.

File: index.html (hunk 1 above)
Comment (PAZHames) [reply index.html]: how weird, i already caught and changed that before I pushed but guess it must not have committed. thanks for the spot!

**Comment by PAZHames** [comment]
typo corrected

**Pull request merged**